manage.py

```diff
@@ -113,4 +113,10 @@ def parse_args():
 
     from django.core.management import execute_from_command_line
 
+    import ptvsd
+    try:
+        ptvsd.enable_attach("my_secret", address = ('0.0.0.0', 21000))
+    except:
+        pass
+
```

Comment:
don't forget to take this out of final PR

Reply:
Yeah I'll make sure this gets removed